cdap-app-fabric/src/test/java/co/cask/cdap/internal/profile/ProfileServiceTest.java

```diff
@@ -205,7 +206,8 @@ public void testProfileService() throws Exception {
     try {
       profileService.deleteProfile(profileId2);
       Assert.fail();
-    } catch (ProfileConflictException e) {
+      // TODO: CDAP-14770 make profileconflictexception extend runtimeexception so the exception can be propagated
```

Comment:
not clear to me what this means. I don't think it should be propagated as runtime exception, it is checked on purpose to let the caller know about the conflict.

Reply:
Good point, ensured the exception was propagated as a ProfileConflictException